app/models/user.rb

```diff
@@ -28,20 +24,18 @@ class User < ActiveRecord::Base
       case_sensitive: false
     }
 
-  scope :active, lambda {
-    not_deleted.where("users.invitation_created_at IS NULL OR users.invitation_accepted_at IS NOT NULL")
-  }
+  scope :active, -> { not_deleted }
   scope :not_deleted, -> { where("deleted_at IS NULL") }
 
   # Overide devise finder to lookup by username or email
-  def self.find_for_database_authentication(warden_conditions)
-    conditions = warden_conditions.dup
-    if login = conditions.delete(:login)
-      where(conditions.to_h).where(["username = :value OR email = :value", { value: login }]).first
-    else
-      where(conditions.to_h).first
-    end
-  end
+  # def self.find_for_database_authentication(warden_conditions)
```

Comment:
Yeah just wanted to keep the SQL around incase we implement login with either email or username with clearance